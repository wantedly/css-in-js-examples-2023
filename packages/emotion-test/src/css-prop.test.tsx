import { render } from "@testing-library/react";
import { test, expect } from "@jest/globals";

import App from "./css-prop";

test("snapshot", () => {
  render(<App />);
  expect(document.children[0].innerHTML).toMatchSnapshot();
});
