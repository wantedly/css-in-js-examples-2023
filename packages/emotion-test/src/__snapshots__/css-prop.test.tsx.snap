// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `"<head></head><body><div><button css="You have tried to stringify object returned from \`css\` function. It isn't supposed to be used directly (e.g. as value of the \`className\` prop), but rather handed to emotion so it can handle it (e.g. as value of \`css\` prop).">Test</button></div></body>"`;
